cogs/clash_river_war.py

```diff
@@ -61,6 +64,43 @@ def boat_attackers_embed(boat_attackers, clan_info):
 
     return embed
 
+def clan_river_race_history_embed(clan_players_war_history: Dict[str, PlayerActivity]):
+    embed = discord.Embed(
+        description=dedent('''
+            Clan River Race History

```

Comment:
Also add in description about how to be ElderWorthy:
```
ElderWorthy means got 1200 fame (minimum) per week over past month. 1200 = doing 3 of 4 war days and losing all of them. i.e. 1200 = 3 x 4 decks x 100 fame per non-boat-attack-loss.
```

Reply:
Also add a note mentioning the order of the FameHistory. How first item is fame from 1 week ago, then 2, 3, 4.

Reply:
And make it pretty format too like what I have done for !ausclan

Reply:
Done